### DISCUSSION TITLE
Rename this project to `ts-semver`

### DISCUSSION THREAD

**Discussion by mrazauskas** [typed-ember/semver#27] [Ideas]
Was it ever considered to name this project as `ts-semver`?

Currently `ts` is used as a suffix and I propose to use it as a prefix. That way `ts` becomes a namespace.

There are many projects using `ts` as prefix. Few examples:

- [`ts-morph`](https://www.npmjs.com/package/ts-morph)
- [`ts-node`](https://www.npmjs.com/package/ts-node)
- [`ts-mocha`](https://www.npmjs.com/package/ts-mocha)
- [`ts-jest`](https://www.npmjs.com/package/ts-jest)

**Comment by chriskrycho** [comment]
Thanks, but I think semver-ts as a GitHub project and repo name is fine, and matches the way it is named in full: Semantic Versioning for TypeScript Types, not TypeScript Types Semantic Versioning. I also don’t want to buy a new domain to maintain forever in parallel with semver-ts.org! Thanks.
